msg/en_US/utils.msg

```diff
@@ -834,7 +834,11 @@ valid options:\n\
                                 6 dump disk bitmaps\n\
                                 7 dump catalog\n\
                                 8 dump log\n\
-                                9 dump heap\n
+                                9 dump heap / class\n\
```

Comment:
This revision is not a final edition and should remain a hidden option until the final version is released. There is even a possibility of changing the specifications as follows to compatible with other utilies.

`cubrid diagdb [OPTION] database-name [class_name1 class_name2 ...]`

One option is to separate the issue of usage output.

Reply:
Rollback usage message  to original `diagdb` usage message.